### PR TITLE
Mobile skeleton

### PR DESCRIPTION
**Pull request by kaitcham**
In this capstone project I created a web app for Kigali Fitness Center
The project consist of two page which both have the desktop version and mobile version:

- The home page have a link in the menu that links it to the about page.
- The logo in the header links to the home page
- The section "Featured speakers" is created dynamically in JavaScript

Please watch the following [Loom Video](https://www.loom.com/share/268f49cb2dd54385b361ff534f9430e7) for more information

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+  color: #d3d3d3;
+  font-size: 20px;
+  text-transform: uppercase;
+}
+
+.footer-container {
+  display: flex;
+  flex-direction: column;
+  padding: 15px 0;
+}
```
Comment (keroloussamy) [inline style.css:259]: - [ ] Kindly add a darker background color for the footer at the about page. It's white on the home page but in the about it's dark.

Your design
image

Original design
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Sequi,
+              vel?
+            </p>
+          </div>
+        </div>
+        <div class="button-container">
+          <button>Register Now</button>
+        </div>
+      </section>
```
Comment (keroloussamy) [inline index.html:99]: - [ ] Kindly add 2 more cards as the design and if you can add some padding between the elements to make the original design.

Your design
image


original design
image

File: index.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  {
+    id: 2,
+    name: 'SohYeong Noh',
+    image: './images/sohyeong.png',
+    alt: 'featured speaker image',
+    intro: 'Nutritionist with over 12years of experience',
+    descr:
+      'He is an experienced nutritionist , Lorem ipsum dolor sit, amet consectetur adipisicing elit. Beatae, modi. Lorem ipsum dolor sit',
+  },
+];
```
Comment (keroloussamy) [inline index.js:43]: - [ ] Kindly add the rest of the speakers you should have 6 cards here you just have 2, add the rest as the original deign.

Your design
image

Original design
image

File: pages/about.html [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+            We also expect the same from them. Our track record speaks for
+            itself.
+          </p>
+          <p>
+            Please contact us per Email for any further questions about Kigali
+            Fitness center
+            <span>kfitnesscenter@gmail.com</span>
+          </p>
+        </div>
+      </section>
```
Comment (keroloussamy) [inline pages/about.html:69]: - [ ] Kindly make you header bigger to match the original design.
- [ ] Kindly center them at the middle of the page.

Your design
image

Original design
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+          <p>
+            Hello! I'm a sports coach for 5 years. I have some master's degree
+            of deadlift, bench press and boxing. Individual approach to each
+            client's it's my job. Also, in addition to training programs, I
+            prescribe a nutrition program for better effect
+          </p>
+          <h3>2021.11.14(SUN) ~ 18(THURS)</h3>
+          <p>@Kigali Fitness Center</p>
+        </div>
+      </section>
```
Comment (keroloussamy) [inline index.html:63]: - [ ] Kindly make you header bigger to match the original design.
- [ ] Kindly center them at the middle of the page.
- [ ] Kindly match the fonts for the items as the original design.


Your design
image

Original design
image

**Review by keroloussamy — APPROVED** [review]
Hi @kaitcham, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**